test/fw/ptl/lib/ptl_sched.py

```diff
@@ -802,6 +802,21 @@ def set_dedicated_time_file(self, filename):
                          str(filename))
         self.dedicated_time_file = filename
 
+    def revert_fairshare(self):
+        """
+        Helper method to revert scheduler's fairshare tree.
+        """
+        cmd = [os.path.join(self.pbs_conf['PBS_EXEC'], 'sbin', 'pbsfs'), '-e']
+        if self.sc_name is not 'default':
+            cmd += ['-I', self.sc_name]
+        self.du.run_cmd(cmd=cmd, runas=self.user)
+        self.parse_sched_config()
+        if self.platform == 'cray' or self.platform == 'craysim':
+            self.add_resource('vntype')
+            self.add_resource('hbmem')
```

Comment:
Are these really reverting fairshare?

Reply:
Well this is the default resource group we go back to on these platforms.

Reply:
oh, you are right, this isn't resource_group, I'll revert this back to where it was.